src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<appengine-web-app xmlns="http://appengine.google.com/ns/1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://kenai.com/projects/nbappengine/downloads/download/schema/appengine-web.xsd appengine-web.xsd">
+    <application>broad-avram-dev</application>
+    <version>1</version>
```

Comment:
Not sure how to handle this. Jenkins deploys should probably deploy to a real version. It would be nice if a local checkout did not have this version configured so we don't accidentally deploy WIP to that real version. I've been editing this locally to something like `gawb-3691` and so far have not forgotten to do that (I think) but it makes me nervous.